backend/site.go

```diff
@@ -55,7 +61,16 @@ func addSite(c *fiber.Ctx) error {
 }
 
 func getRootSites(c *fiber.Ctx) error {
-	cursor, err := sites.Find(c.Context(), bson.D{{"parent", nil}})
+	user, ok := c.Locals("userID").(string)
+	if !ok {
+		return c.Status(500).SendString("Invalid userID in header")
+	}
+	userID, err := primitive.ObjectIDFromHex(user)
+	if err != nil {
+		return c.Status(500).SendString(err.Error())
+	}
+
+	cursor, err := sites.Find(c.Context(), bson.D{{"parent", nil}, {"owner", userID}})
```

Comment:
I let those linter errors pass because the Key-Value -notation is really cumbersome to write, and most of the time they are used to interact with MongoDB-driver, which _should_ not break. They are an eyesore though, so here they are fixed:
https://github.com/Slowers-Team/Slowers-App/pull/47/commits/a02597e4473b63d9f2662a8f9dda11c5e24aaff8